Purchases/Networking/HTTPClient.swift

```diff
@@ -75,6 +75,7 @@ private extension HTTPClient {
             "X-Platform-Flavor": systemInfo.platformFlavor,
             "X-Client-Version": SystemInfo.appVersion,
             "X-Client-Build-Version": SystemInfo.buildVersion,
+            "X-StoreKit2-Enabled": self.systemInfo.useStoreKit2IfAvailable ? "true" : "false",
```

Comment:
I think this could also be
```swift
"X-StoreKit2-Enabled": "\(self.systemInfo.useStoreKit2IfAvailable)"
```
?
I don't have a preference though 😂 they're equally readable to my eyes

Reply:
I didn't know `Bool` had that as the `CustomStringConvertible` implementation 😅 I'll change it.